subjects/issamestring/README.md

```diff
@@ -0,0 +1,43 @@
+## issamestring
+
+### Instructions
+
+Write a function that returns `true` if the two `strings` passed as parameter are equal. Otherwise it returns `false`.
+
+The function will ignore the case of the characters.
+
+### Expected function
+
+```go
+func IsSameString(s1, s2 string) bool {
+
+}
+```
+
+### Usage
+
+Here is a possible program to test your function :
```

Comment:
Remove space in between 'function' and ':'

Reply:
fixed